src/services/character/character.service.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'
+import { Query } from 'mongoose'
+import characters from '../../data/models/character-schema'
+import { CharacterType } from '../../types/character/character-type'
+
+export class CharacterService {
+  private static _instance: CharacterService
+
+  public static getInstance(): CharacterService {
+    if(_.isNil(CharacterService._instance)) {
+      CharacterService._instance = new CharacterService()
+    }
+    return CharacterService._instance
+  }
+
+  public async getEntity(id: string) {
+    return await characters.findOne({ ownerId: id }, (err: Error, characterFound) => {        
```

Comment:
Could you try to remove the return and test the functionality?
I am not sure that the return is required but if you could test it for me it will be nice!

Reply:
I'll try to do!

Reply:
Typescript says: an async/await function must return a value of type "Promise<T>" if I remove the return.

Reply:
ok this make sense